ops/scripts/create_environment_migrate.sh

```diff
@@ -0,0 +1,127 @@
+# This script sets up a completely new Cloud.gov CF Space with all the corresponding
```

Comment:
The one thing that tripped me up when creating Dave's sandbox (and I should've updated the create_sandbox script's comment in hindsight): I ran the script while on a feature branch and docker was composed up, so the build command failed and the created branch did not originate from main. Worth adding to the comments: "Run from main with docker composed down"?